src/components/attendee/Banner/Banner.module.scss

```diff
@@ -0,0 +1,36 @@
+.banner {
+  position: fixed;
+  z-index: 90;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  opacity: 1;
+
+  transition: opacity 1.5s ease;
+  transition-delay: 1s;
+  height: 100vh;
+  width: 100vw;
+}
+
+.bannerContent {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  color: white;
```

Comment:
color constant here please

Reply:
Fixed in https://github.com/sparkletown/sparkle/pull/2952/commits/31283fe4933ee54a28e2a97af203f53d7c7886bf